[docs] Changed information about publicDomainTemplate at documentation (#13897)
deckhouse/deckhouse: docs/documentation/pages/DECKHOUSE_CONFIGURE_GLOBAL.md

The global Deckhouse settings are stored in the `ModuleConfig/global` resource (see [Deckhouse configuration](./#deckhouse-configuration)).

{% alert %}
The [publicDomainTemplate](#parameters-modules-publicdomaintemplate) parameter specifies a DNS name template used by some Deckhouse modules to create Ingress resources. If this parameter is not specified, Ingress resources will not be created.

If you don't have access to wildcard DNS records, you can use [sslip.io](https://sslip.io) or similar services for testing purposes.

> The domain specified in the template must not match or be a subdomain of the domain specified in the [`clusterDomain`](https://deckhouse.io/documentation/v1/installing/configuration.html#clusterconfiguration-clusterdomain) parameter. We do not recommend changing the `clusterDomain` value unless absolutely necessary.

Reviewer: ```suggestion
The domain specified in the template must not match or be a subdomain of the domain specified in the [`clusterDomain`](https://deckhouse.io/documentation/v1/installing/configuration.html#clusterconfiguration-clusterdomain) parameter. It is not recommended to change the `clusterDomain` value unless absolutely necessary.
```

> For the template to work correctly, DNS services must be properly configured both in the networks where the cluster nodes reside and in the networks from which users will access the platform’s web interfaces.

Reviewer: ```suggestion
For the template to work correctly, you must first configure DNS services both in the networks where the cluster nodes will be located and in the networks from which access to the service web interfaces is required.
```
